includes/class-reader-activation.php

```diff
@@ -935,12 +936,23 @@ public static function process_auth_form() {
 			return;
 		}
 		$action   = isset( $_POST['action'] ) ? \sanitize_text_field( $_POST['action'] ) : '';
-		$email    = isset( $_POST['email'] ) ? \sanitize_email( $_POST['email'] ) : '';
+		$email    = isset( $_POST['npe'] ) ? \sanitize_email( $_POST['npe'] ) : '';
 		$password = isset( $_POST['password'] ) ? \sanitize_text_field( $_POST['password'] ) : '';
 		$redirect = isset( $_POST['redirect'] ) ? \esc_url_raw( $_POST['redirect'] ) : '';
 		$lists    = isset( $_POST['lists'] ) ? array_map( 'sanitize_text_field', $_POST['lists'] ) : [];
+		$honeypot = isset( $_POST['email'] ) ? \sanitize_email( $_POST['email'] ) : '';
```

Comment:
```suggestion
		$honeypot = isset( $_POST['email'] ) ? \sanitize_text_field( $_POST['email'] ) : '';
```

We're not interested in sanitizing this input as email. It would result in an empty string on an invalid email and let the bot in.

`sanitize_text_field()` should suffice.

Reply:
You're right—we want to trap _any_ value in the field, not just email addresses!